### ISSUE TITLE
Scan BCSC v4 for missing a11y labels and test IDs

### ISSUE THREAD

**Issue by bryce-mcmath** [bcgov/bc-wallet-mobile#3186]
### Description

Scan through BCSC v4 codebase for missing `testID` and `accessibilityLabel` on all interactive elements and add them where they are missing. This includes:
- Buttons
- TouchableOpacitys
- Pressables
- RadioButtons
- Switches
- Toggles
- Inputs

Missing testIDs prevents us from writing effective e2e test suites and missing accessibility labels means we will later fail our accessibility audit.

#### Specifics Screens:
- Create PIN
- Enter PIN
- Login/First Screen: The testIDs are based on the account nickname (`CardButton-Myself)`, shouldn't it be something more agnostic like `CardButton-0`
- back (arrow) buttons
- Hamburger Menu Button

**Comment by jleach** [comment]
Fixed in 4.0.0-4297.

**Comment by bryce-mcmath** [comment]
Fixed
